### PR TITLE
Squash commits on setup

### LINKED ISSUE #15: Clear thoughbot commits after run setup script
Since the proteus user is creating a new project from a template, which is this repository, it wouldn't make sense to clear all commits from this repository after run the setup bash script?

### PR DESCRIPTION
**Pull request by joshuaogle**
Fixes #15. Also removed the `-e` flags for the echo commands since they strings don't need escaping.

### CODE REVIEW

File: bin/setup [hunk 1]
```diff
@@ -2,20 +2,24 @@
 
 # Run this script immediately after cloning the codebase.
 
+# Squash all commits to get a clean slate
+git update-ref -d refs/heads/master
+git commit -am "New Proteus-Middleman project"
+
 # Make sure Bundler is installed
 if [ `gem query -i -n bundler` = "false" ]; then
```
Comment (gabebw) [inline bin/setup]: The `-n` isn't necessary, I don't think - it's the default.

File: bin/setup [hunk 2]
```diff
@@ -2,20 +2,24 @@
...
+echo "Installing Dependencies..."
 bundle install
 
 # Remove Git remote if it's still the proteus-middleman repo
 if [ `git config --get remote.origin.url` = "git@github.com:thoughtbot/proteus-middleman.git" ]; then
   git remote rm origin
-  echo -e "Resetting Git remote. What is your repo url?"
+  echo "Resetting Git remote. What is your repo url?"
   read url
   git remote add origin $url
```
Comment (gabebw) [inline bin/setup]: Always quote variables: `"$url"`

File: bin/setup (hunk 1 above)
Comment (joshuaogle) [inline bin/setup]: The docs made it sound like it wasn't a default, but it does seem to work exactly the same without it. :+1: Fixed

**Comment by joshuaogle** [comment]
Thanks @gabebw! I also switched the backticks to `$()` since ShellCheck says the backticks are deprecated. Still works fine as far as I can tell

**Comment by gabebw** [comment]
Excellent. Shellcheck is great.

File: bin/setup [hunk 3]
```diff
@@ -2,20 +2,24 @@
...
 fi
 
 # Set up Ruby dependencies via Bundler
-echo -e "Installing Dependencies..."
+echo "Installing Dependencies..."
 bundle install
 
 # Remove Git remote if it's still the proteus-middleman repo
-if [ `git config --get remote.origin.url` = "git@github.com:thoughtbot/proteus-middleman.git" ]; then
+if [ $(git config --get remote.origin.url) = "git@github.com:thoughtbot/proteus-middleman.git" ]; then
```
Comment (gabebw) [inline bin/setup]: Actually, this should be quoted too (I know right): `"$(git config --get remote.origin.url)"`

File: bin/setup [hunk 4]
```diff
@@ -2,20 +2,24 @@
...
 # Run this script immediately after cloning the codebase.
 
+# Squash all commits to get a clean slate
+git update-ref -d refs/heads/master
+git commit -am "New Proteus-Middleman project"
+
 # Make sure Bundler is installed
-if [ `gem query -i -n bundler` = "false" ]; then
-  echo -e "Installing Bundler..."
+if [ $(gem query -i -n bundler) = "false" ]; then
```
Comment (gabebw) [inline bin/setup]: Best practice is to put quotes around `$()`

File: bin/setup (hunk 3 above)
Comment (joshuaogle) [inline bin/setup]: That makes sense. Done.

**Comment by gabebw** [comment]
:+1: Looks great!

**Pull request closed without merging**